core/src/main/java/io/questdb/cairo/arr/ArrayView.java

```diff
@@ -463,6 +464,10 @@ public final long getVanillaMemoryLayoutSize() {
                 + getCardinality() * elemSize; // data
     }
 
+    public final Utf8Sequence getVarchar(int flatIndex) {
```

Comment:
lets document semantics of this method, if it returns mutable/reusable object or objects that can be cached externally. Otherwise this is ambiguous and we're continuing to create strings